Misc/NEWS.d/next/Tests/2024-05-04-22-56-41.gh-issue-101525.LHK166.rst

```diff
@@ -0,0 +1 @@
+Skip test_gdb if the binary is BOLTed. Patch by Donghee Na.
```

Comment:
I suggest "is linked with BOLT" rater than "is BOLTed".

Reply:
While I like the sound of "BOLTed", I would also prefer Victor's suggestion. By the way, you could use <code>\`\`test_gdb\`\`</code> instead, just to make it stand a bit more for readers.